ilastik/shell/gui/startShellGui.py

```diff
@@ -73,6 +76,13 @@ def _applyStyleSheet(app):
         app.setStyleSheet(styleSheetText)
 
 
+def getSplashScreen():
+    splash_path = os.path.join(os.path.split(ilastik.__file__)[0], 'ilastik-splash.png')
```

Comment:
This is probably bikeshedding, but
```suggestion
    splash_path = os.path.join(os.path.dirname(ilastik.__file__), 'ilastik-splash.png')
```